### ISSUE TITLE
🚀 FEATURE: Improve configurability by shipping with a neobrutal-config.json

### ISSUE THREAD

**Issue by adriankarlen** [adriankarlen/neobrutal-zebar#30]
User needs have shown that a lot of users want to use Neobrutal as a base, but with a lot of additional configurations (whole bar), different order of elements etc. It would be nice if a user could have more configurability for this without requiring to write actual code.

Current ideas of configurable options include:

- whole bar / segmented
- predefined themes (eg. theme = "catppuccin mocha" -> would apply the mocha recipe)
  - config.css would still need to be master here (to allow user overrides)
- workspace icons (index, display_name or custom)
- order of items (eg. be able to place workspaces in left group and now playing in center group)

**Comment by jakob-nova** [comment]
Big like on the workspace icons idea. I love your config and being able to have the workspace index displayed would really help for multi-screen setups.

**Comment by Entharia** [comment]
Definitely! Sounds like we really need this. Looks like I won't have to fork it after all if this is in the works now :)
